tests/issue_1512/TarCpUntar.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
Please apply the coding guidelines also to tests :-)
More importantly,  how is that code related to testing the #1512?   It looks like you test the tarfile module - that is not our business...